linux/usage/cron.md

```diff
@@ -39,10 +39,10 @@ The layout for a cron entry is made up of six components: minute, hour, day of m
 For example:
 
 ```
-0 0 * * *  /home/pi/backup.sh
+15 0 * * *  sh /home/pi/backup.sh
```

Comment:
I think the `sh` is only needed if your script file doesn't have it's executable bit set? (i.e. you haven't done `chmod +x backup.sh` yet)

Reply:
You are absolutely right, same on the comment below. I will recall the request, update and put in a new one. Thanks!